### DISCUSSION TITLE
Which dataset is selected for training

### DISCUSSION THREAD

**Discussion by cantaible** [rehanguha/brisque#6] [Q&A] [answered]
Hi, thank you very much for providing this useful toolbox. Could you give more detail on how to trian the SVM in the last step to get the brisque score? Which dataset your select for training? Can I use your trained weight file on my customed data? Thanks.

**Comment by rehanguha** [comment, marked as answer]
Hi,
Yes generally you can use this trained file for your images and it works best for natural images.

Here the custom model is from Tampere Image Database (TID2008) ::  [Paper](https://ponomarenko.info/papers/mre2009tid.pdf), [Dataset](https://www.ponomarenko.info/tid2008.htm)

In terms of the model it is SVR with Radial Basis Function Kernel. I have used the model provided by the authors. [Model Link](https://github.com/dsoellinger/blind_image_quality_toolbox/tree/master/%2Bbrisque)

If you are willing please contribute the script for training the SVM model. We can brainstorm if needed. 

P.S. - I am moving this to discussion as it will be benefited by others.

Similar Discussion: https://github.com/rehanguha/brisque/discussions/7
